.azure-pipelines.yml

```diff
@@ -145,6 +210,53 @@ jobs:
   - template: tools/ci/azure/tox_pytest.yml
     parameters:
       directory: tools/wpt/
+      toxenv: py27
+
+- job: wpt_integration_macOS_py3
+  displayName: 'tools/wpt/ tests: macOS (Python 3.6)'
+  dependsOn: decision
+  condition: dependencies.decision.outputs['test_jobs.wpt_integration']
+  pool:
+    vmImage: 'macOS-10.14'
+  steps:
+  # full checkout required
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '3.6.x'
+  - task: UsePythonVersion@0
+    inputs:
+      versionSpec: '2.7.x'
```

Comment:
Was it a copy-paste error to install 3.6 and then 2.7 directly after?

Reply:
I figured it out, will make it clearer in https://github.com/web-platform-tests/wpt/pull/21724.